Cargo.toml

```diff
@@ -58,6 +58,7 @@ std = [
     "winter-crypto/std",
     "winter-math/std",
     "winter-utils/std",
+    "getrandom/js",
```

Comment:
Why do we need to add this here? I would have thought that we'd need something like this for `no-std` build rather than `std`.

Reply:
That's a leftover from a previous attempt at finding why the CI failed. Removed now